src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -139,10 +140,14 @@ add_executable(
   ${PERF_ANALYZER_UNIT_TESTS_HDRS}
   mock_inference_profiler.h
   test_inference_profiler.cc
+  test_command_line_parser.cc
   test_model_parser.cc
   $<TARGET_OBJECTS:json-utils-library>
 )
 
+set_target_properties(perf_analyzer_unit_tests 
+  PROPERTIES COMPILE_FLAGS "-Wno-write-strings")
```

Comment:
Because without, I cannot statically create my input argv cases in the unit tests without that warning popping up hundreds of times for each compilation. Since this is only targeting the unit test and not the production binary, it should be fine.

i.e., I cannot do...
```
char *argv[argc] = {"name". "--opt1", "val1", "--opt2", "val2"};
```
Writing strings like that is not recommended in C++, hence the `-Wwrite-strings` warning. I am disabling it so I can make my unit tests easier to write.

Reply:
I see. Thanks for the response! Can you add a comment for the same?

Reply:
sure